cmd/util.go

```diff
@@ -1,22 +1,82 @@
 package cmd
 
 import (
+	"context"
 	"os"
+	"time"
 
-	"github.com/spf13/cobra"
-
+	"github.com/cloudquery/cloudquery/internal/logging"
+	"github.com/cloudquery/cloudquery/internal/signalcontext"
+	"github.com/cloudquery/cloudquery/internal/telemetry"
+	"github.com/cloudquery/cloudquery/pkg/ui"
 	"github.com/cloudquery/cloudquery/pkg/ui/console"
+
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
 )
 
-func handleError(f func(cmd *cobra.Command, args []string) error) func(cmd *cobra.Command, args []string) {
+func handleError(f func(context.Context, *console.Client, *cobra.Command, []string) error) func(cmd *cobra.Command, args []string) {
```

Comment:
I liked this fix, makes command much cleaner, not sure handleError is correct anymore since it does more, maybe handleCommand / executeCommand, you add the handle error inside this function, or just keep it as is in the current function.

Reply:
it's now `handleCommand`